src/aleph_client/asynchronous.py

```diff
@@ -64,9 +64,9 @@ def get_fallback_session() -> ClientSession:
 
 
 async def ipfs_push(
-    content,
-    session: Optional[ClientSession] = None,
-    api_server: str = settings.API_HOST,
+        content,
+        session: Optional[ClientSession] = None,
+        api_server: str = settings.API_HOST,
```

Comment:
Please use https://black.readthedocs.io/en/stable/ instead for auto-formatting code.